extensions/ql-vscode/src/model-editor/model-editor-view.ts

```diff
@@ -290,12 +290,12 @@ export class ModelEditorView extends AbstractWebview<
         void telemetryListener?.sendUIInteraction("model-editor-switch-modes");
 
         break;
-      case "hideModeledApis":
-        this.hideModeledApis = msg.hideModeledApis;
+      case "hideModeledMethods":
+        this.hideModeledMethods = msg.hideModeledMethods;
         await this.updateMethodsUsagePanelState(
           this.methods,
           this.databaseItem,
-          this.hideModeledApis,
+          this.hideModeledMethods,
         );
         void telemetryListener?.sendUIInteraction(
           "model-editor-hide-modeled-apis",
```

Comment:
Do we care about changing the telemetry interaction to `methods` too?

Reply:
Good spot. I think we should also change that. Make everything consistent before we start generating telemtry from external users of the model editor.